index.html

```diff
@@ -1913,6 +1919,67 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    When accessing data or requesting permission, [=sites=] (and other [=actors=]) should provide
+    [=people=] with relevant information about its use and [=user agents=] should help present and
+    consume that information.
+  </span>
+</div>
+
+Transparency is a necessary, but insufficient, condition for [=consent=]. Relevant information
+includes who is accessing information, what information is accessed (including the potential
```

Comment:
Here we have two different meanings of "information" in the same sentence. First the information/explanation that needs to be given to a user, and second the information whose use the user is being informed about.

Would it help to call this a "helpful explanation" instead of "relevant information"?

Reply:
Agree that this was ambiguous over "information". Thinking about phrasing, but "relevant explanation", "explanatory text", something like that.